src/Windowing/Silk.NET.Windowing.Glfw/GlfwWindow.cs

```diff
@@ -306,16 +307,29 @@ protected override void CoreInitialize(WindowOptions opts)
             // Set video mode (-1 = don't care)
             _glfw.WindowHint(WindowHintInt.RefreshRate, opts.VideoMode.RefreshRate ?? -1);
             _glfw.WindowHint(WindowHintInt.DepthBits, opts.PreferredDepthBufferBits ?? -1);
+            _glfw.WindowHint(WindowHintInt.StencilBits, opts.PreferredStencilBufferBits ?? -1);
```

Comment:
I don't know, but are you sure we should set the hint if the user provided no values?

Reply:
-1 is GLFW_DONT_CARE meaning that is has the same effect as not setting it at all